inc/command/class-command.php

```diff
@@ -1054,7 +1081,270 @@ protected function merge_config( array $config, array $overrides ) : array {
+		} else {
+
+			$base_command = sprintf( 'cp "%s" "%s" &> /dev/null', $template_file, $target_file );
+			passthru( $base_command, $return_var );
+
+			if ( $return_var ) {
+				$output->writeln( '<error>Could not create markdownlint configuration file, exiting.</error>' );
+
+				return 1;
+			}
+		}
+
+		// Copy styles folder for Vale
+		$target_folder = getcwd() . '/styles';
+		$template_folder = realpath( __DIR__ . '/../../templates/docslint/styles' );
+
+		if ( file_exists( $target_folder ) ) {
+			$output->writeln( '<error>Documentation linter Style folder already exists at <root>/styles, not overwriting.</error>' );
+		} else {
+
+			$base_command = sprintf( 'cp -r "%s" "%s" &> /dev/null', $template_folder, $target_folder );
+			passthru( $base_command, $return_var );
+
+			if ( $return_var ) {
+				$output->writeln( '<error>Could not create documents linter styles folder, exiting.</error>' );
+
+				return 1;
+			}
+		}
+
+		// Copy configuration file for Vale
+		$target_file = getcwd() . '/.vale.ini';
+		$template_file = realpath( __DIR__ . '/../../templates/docslint/.vale.ini' );
+
+		if ( file_exists( $target_file ) ) {
+			$output->writeln( '<error>Documentation linter configuration file already exists at <root>/.vale.ini, not overwriting.</error>' );
+		} else {
+
+			$base_command = sprintf( 'cp "%s" "%s" &> /dev/null', $template_file, $target_file );
+			passthru( $base_command, $return_var );
+
+			if ( $return_var ) {
+				$output->writeln( '<error>Could not create documents linter configuration file, exiting.</error>' );
+
+				return 1;
+			}
+		}
+
+		$output->writeln( '<info>Files have been copied to the root of your project: .markdownlint.jasonc, styles/, and .vale.ini.</info>' );
+		$output->writeln( '<info>Feel free to edit the generated config files to install Style guides or configure as needed.</info>' );
+
+		return 0;
+	}
+
+
+	/**
+	 * Runs Documentation Linting with default config by default.
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 *
+	 * @return int
+	 */
+	protected function lintdocs( InputInterface $input, OutputInterface $output ) {
+		$subcommands = [ 'files', 'markdown', 'style' ];
+		$options = $input->getArgument( 'options' );
+		$subsubcommand = $options[0] ?? null;
+
+		$lint_path = $input->getOption( 'lint-path' ) ?? '.';
+		$lint_path = rtrim( $lint_path, '\\/' );
+
+		if ( ! in_array( $subsubcommand, $subcommands, true ) && ( $subsubcommand !== 'all' ) ) {
+			$output->writeln( sprintf( 'Available sub commands are: %s.', implode( ', ', $subcommands ) ) );
+
+			return 1;
+		}
+
+		// Check path exists
+		if ( ! file_exists( $lint_path ) ) {
+			$output->writeln( sprintf( '<error>Docslint cannot find the path you specified "%s", exiting.</error>', $lint_path ) );
+
+			return 1;
+		}
+
+		if ( $subsubcommand === 'all' ) {
+			foreach ( $subcommands as $subsubcommand ) {
```

Comment:
Not a big fan of the naming here, plus you're overwriting `$subsubcommand` in the loop here, probably unintentionally, but may come and bite us later on if we decided to use `$subsubcommand` later on and expect it to be `all`. I think naming it what it represents would be easier to understand, like `context` or `scope`.

It's also not clear to me why you have subcommands that represent scopes, and then an additional subsubcommand that alters the scope to include all of them. I feel like this can be simplified a lot.